magento/utils/client/types.ts

```diff
@@ -0,0 +1,183 @@
+export interface CategoryLink {
+  position: number;
+  category_id: string;
+}
+
+interface Image {
+  url: string;
+  code: string;
+  height: number;
+  width: number;
+  label: string;
+  resized_width: number;
+  resized_height: number;
+}
+
+export interface CustomAttribute {
+  attribute_code: string;
+  value: string | number | string[];
+}
+
+export interface Categoria {
```

Comment:
Use o nome das coisas em ingles. Se for uma tipagem do Magento, pode adicionar isso como prefixo `MagentoCategory